src/show.c

```diff
@@ -41,22 +41,25 @@ static void print_cfl(unsigned int N, const long dims[N], const complex float* d
 
 	long T = md_calc_size(N, dims);
 
-	const char* allowed_fmts[] = {
-
-		"%+e%+ei",
-		"%+f%+fi",
+	char* allowed_fmts[] = {
+	 	"%%%[+0-9.]f%%%[+0-9.]fi",
+		"%%%[+0-9.]e%%%[+0-9.]ei"
```

Comment:
Think we should require the sign (at least between real and imaginary part),
i.e. something lik this: %%%[+0-9.]fi -> %%+%[+0-9.]fi
What do you think?

(Also thank you for fixing the white space errors, but the empty line after the 
for (..) { and struct {  is part of the style I use in BART.)